test/langtools/tools/javac/processing/model/util/elements/MethodParameters/ClassContainer.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I recommend folding the enum being processed into the source of ImplicitParametersProcessor.java. The jtreg tags can then first compile the source of the processor and then secondly run the processor on itself. This technique is commonly used in annotation processing tests.